docs/validation.md

```diff
@@ -0,0 +1,68 @@
+# Validating BuildKit CLI for kubectl
+
+While this CLI can be used for CI usecases where the images are always pushed to a registry, the primary
+usecase is for developer scenarios where the image is immediately available within the Kubernetes cluster.
+The example listed here can be used to validate everything is set up properly and you're able to build images
+and run pods with those images.
+
+## Remove any prior default builder
+```
+kubectl buildkit rm buildkit
+```
+
+## Build a simple image
+This image just emits a single line of text output and exits with success - suitable to run as a Job.
+Note that we are not using the tag "latest" as that changes Kubernetes behavior and forces a pull from the registry.
+```
+cat << EOF | kubectl build -t acme.com/some/image:1.0 -f - .
+FROM busybox
+ENTRYPOINT ["echo", "Success!"]
+EOF
+``` 
+
+## Run a Job with the image we just built
+```
+cat << EOF | kubectl apply -f -
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: testjob1
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: jobc
+        image: acme.com/some/image:1.0
+        imagePullPolicy: Never
+EOF
+```
+
+## Confirm Success
+```
+kubectl get job testjob1
+```
+You want to see `COMPLETIONS` showing `1/1`
+
+If not, then troubleshoot below...
+
+## Troubleshooting
+```
+kubectl logs -l job-name=testjob1
+
+kubectl describe pod -l job-name=testjob1
+```
+
+If the images are not getting loaded properly into the container runtime, you'll see an Event that looks like this:
```

Comment:
This was a little confusing because there are the two commands above. I'm not sure if the warning will also show up in the logs, or only when you describe the job.